### PR TITLE
fix(multiple): replace all usages of color grey with granite

### PR DESCRIPTION
**Pull request by mhaertwig**

### CODE REVIEW

**Review by kyubisation — APPROVED** [review]
LGTM 👍

File: src/angular/styles/_typography.scss [hunk 1]
```diff
@@ -95,17 +95,13 @@
   --sbb-color-lemon: #{sbb.$sbbColorLemon};
   --sbb-color-green: #{sbb.$sbbColorGreen};
 
-  // Non-Standard Colors
-  --sbb-color-grey: #{sbb.$sbbColorGrey};
-  --sbb-color-grey-medium: #{sbb.$sbbColorGreyMedium};
-
```
Comment (jeripeierSBB) [inline src/angular/styles/_typography.scss:101]: This is a breaking change for me which should we better not commit right now. Maybe also leave it here forever.

File: src/angular/styles/_variables.scss [hunk 2]
```diff
@@ -84,10 +84,6 @@ $sbbColorPeach: #fcbb00;
 $sbbColorLemon: #ffde15;
 $sbbColorGreen: #00973b;
 
-// Non-Standard Colors
-$sbbColorGrey: #666666;
-$sbbColorGreyMedium: #979797;
-
```
Comment (jeripeierSBB) [inline src/angular/styles/_variables.scss:90]: This is a breaking change for me which should we better not commit right now. Maybe also leave it here forever.

File: src/angular/styles/_typography.scss (hunk 1 above)
Comment (mhaertwig) [reply src/angular/styles/_typography.scss]: Good point, thanks! I agree with leaving it there forever

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**